geodatasets/lib.py

```diff
@@ -118,6 +119,30 @@ def query_name(self, name: str) -> Dataset:
 
         raise ValueError(f"No matching item found for the query '{name}'.")
 
+    def filter_by_geometry(self, geometry_type: str) -> dict[str, Dataset]:
+        """Return dictionary of :class:`Dataset` objects filtered by geometry type
+
+        Returns dictionary of :class:`Dataset` objects from the :class:`Bunch`
+        filtered by `geometry_type` if found in the :list:`GEOMETRY_TYPES`
+
+        Parameters
+        ----------
+        geometry_type : str
+            GeometryType.name of the data item. Case does not matter.
+
+        Returns
+        -------
+         bunch_filtered : dict
+            dictionary of :class:`Dataset` objects
+        """
+        geometry_type_clean: str = geometry_type.translate(QUERY_NAME_TRANSLATION).upper()
+        if geometry_type_clean not in GEOMETRY_TYPES:
+            raise ValueError(f"No matching item found for the geometry_type '{geometry_type}'.")
```

Comment:
```suggestion
            raise ValueError(f"No matching item found for the geometry_type '{geometry_type}'. Use one of {GEOMETRY_TYPES}")
```

It is always good to give a hint on what the solution shall be.